### DISCUSSION TITLE
建议能不能增加图床的api接口

### DISCUSSION THREAD

**Discussion by wudingjian** [usememos/memos#3070] [Ideas]
现在每次上传图片，都存在数据库中，数据库db文件很大。

能不能增加简单图床的api接口：https://github.com/icret/EasyImages2.0
![简单图床 - EasyImage](https://img.188181.xyz/i/2024/03/10/12yrwli-0.webp)

有浏览器插件：https://github.com/icret/EasyImage-Browser-Extension
![](https://img.188181.xyz/i/2024/03/11/mhlq-0.webp)
配置界面：
![](https://img.188181.xyz/i/2024/03/11/4hn01-0.webp)

或者支持其他图床
支持对图片做一些压缩

**Comment by Coisini-H** [comment]
https://postimages.org/login/api

希望支持这个
